base/db/R/get.trait.data.pft.R

```diff
@@ -17,11 +17,12 @@
 ##' @param dbcon database connection
 ##' @param forceupdate set this to true to force an update, auto will check to see if an update is needed.
 ##' @param trait.names list of trait names to retrieve
+##' @param update logical: Rerun the db if result files already exist?
 ##' @return updated pft with posteriorid
 ##' @author David LeBauer, Shawn Serbin, Rob Kooper
 ##' @export
 get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon, trait.names,
-                               forceupdate = FALSE) {
+                               forceupdate = FALSE, update = FALSE) {
```

Comment:
What's the difference between `forceupdate` and `update` here? Should they be combined into one argument?

Reply:
I think both of them are different.

Reply:
@nanu1605 I can't find anywhere in this function where `update` is used other than where you added it to `query.priors`, which as noted there seems to be an unnecessary addition. I think you can remove the `update` argument and usage in query.priors